miniapps/tools/load-visit-dc.cpp

```diff
@@ -0,0 +1,116 @@
+// Copyright (c) 2010, Lawrence Livermore National Security, LLC. Produced at
+// the Lawrence Livermore National Laboratory. LLNL-CODE-443211. All Rights
+// reserved. See file COPYRIGHT for details.
+//
+// This file is part of the MFEM library. For more information and source code
+// availability see http://mfem.org.
+//
+// MFEM is free software; you can redistribute it and/or modify it under the
+// terms of the GNU Lesser General Public License (as published by the Free
+// Software Foundation) version 2.1 dated February 1999.
+//
+//    -------------------------------------------------------------------
+//    Load DC Miniapp:  Visualize fields saved via DataCollection classes
```

Comment:
This is a really nice tool, great idea to add it @v-dobrev!

Following @cyrush's proposal in https://github.com/mfem/mfem/pull/394#issuecomment-353206567 I suggest that we rename it to `load-dc` and just mention that currently only `VisItDataCollection` is supported.

Alternatively, we can consider naming it `dc-explorer` and maybe merge it with `convert-dc`, https://github.com/mfem/mfem/blob/task/2017_12_conduit_support/miniapps/tools/convert-dc.cpp, to support `DataCollection` visualization, conversion, etc. similar to `mesh-explorer`?

Reply:
I'll just rename it to `load-dc` for now. We'll revisit this when reviewing #394.